src/Toolkit.Forms/UtilityNetworkTraceTool/UtilityNetworkTraceTool.cs

```diff
@@ -0,0 +1,766 @@
+            set => SetValue(StartingPointSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated multipoint trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated multipoint trace result.
+        /// </value>
+        public Symbol? ResultPointSymbol
+        {
+            get => GetValue(ResultPointSymbolProperty) as Symbol;
+            set => SetValue(ResultPointSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated polyline trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated polyline trace result.
+        /// </value>
+        public Symbol? ResultLineSymbol
+        {
+            get => GetValue(ResultLineSymbolProperty) as Symbol;
+            set => SetValue(ResultLineSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a <see cref="Symbol"/> that represents an aggregated polygon trace result.
+        /// </summary>
+        /// <value>
+        /// A <see cref="Symbol"/> that represents an aggregated polygon trace result.
+        /// </value>
+        public Symbol? ResultFillSymbol
+        {
+            get => GetValue(ResultFillSymbolProperty) as Symbol;
+            set => SetValue(ResultFillSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the GeoView should automatically zoom to show trace results.
+        /// </summary>
+        public bool AutoZoomToTraceResults
+        {
+            get => (bool)GetValue(AutoZoomToTraceResultsProperty);
+            set => SetValue(AutoZoomToTraceResultsProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the layout size that would be appropriate for the current UI state.
+        /// </summary>
+        /// <remarks>
+        /// On mobile devices, it is often appropriate to minimize the control to show more of the map while adding starting points.
+        /// </remarks>
+        public ElementLayoutSizePreference RequestedLayoutSize
+        {
+            get => (ElementLayoutSizePreference)GetValue(RequestedLayoutSizeProperty);
+            private set => SetValue(RequestedLayoutSizePropertyKey, value);
+        }
+
+        internal static readonly BindablePropertyKey RequestedLayoutSizePropertyKey =
+            BindableProperty.CreateReadOnly(nameof(RequestedLayoutSize), typeof(ElementLayoutSizePreference), typeof(UtilityNetwork), ElementLayoutSizePreference.NotSet);
+
+        /// <summary>
+        /// Identifies the <see cref="RequestedLayoutSize"/> bindable property.
+        /// </summary>
+        public static readonly BindableProperty RequestedLayoutSizeProperty = RequestedLayoutSizePropertyKey.BindableProperty;
```

Comment:
You should declare it readonly. See https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.bindableproperty.isreadonly?view=xamarin-forms#xamarin-forms-bindableproperty-isreadonly